### DISCUSSION TITLE
Welcome to mongoose Discussions!

### DISCUSSION THREAD

**Discussion by vkarpov15** [Automattic/mongoose#9740] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SsekandiRaymond** [comment]
Big fan of mongoose for about 2 years now. Mongoose's `.lean()` is powerful, got to admit it. I mean we all use it quick results, right? However, it is not satisfying enough. I want to just get the query results real quick and respond to the request. In my front-end (react.js and react native), I wanted to use the mongodb _id as key prop value for .jsx or in react native FlatList keyExtractor. It was quite repetitive and tiresome to have to convert the ObjectId to string to use in my front-end. Wrote this extension [mongoose-lean-extension](https://www.npmjs.com/package/mongoose-lean-extension) to ease my work. Try it out...

**Reply by SsekandiRaymond** [reply]
# [MONGOOSE-LEAN-EXTENSION](https://www.npmjs.com/package/mongoose-lean-extension)
Try out my plugin for mongoose, and contribute
